core/src/main/scala/org/elasticmq/actor/queue/QueueActorMessageOps.scala

```diff
@@ -22,6 +22,7 @@ trait QueueActorMessageOps extends Logging {
         receiveMessages(visibilityTimeout, count, receiveRequestAttemptId)
       case DeleteMessage(deliveryReceipt) => deleteMessage(deliveryReceipt)
       case LookupMessage(messageId)       => messageQueue.byId.get(messageId.id).map(_.toMessageData)
+      case MoveMessageToDLQ(message)      => moveToDeadLetterQueue(message)
```

Comment:
maybe simply: `MoveMessage`. Here the fact that we're moving to the DLQ is not relevant (it's relevant at the time when we make the decision, whether to create a new message or move it)

Reply:
indeed, changed to `MoveMessage`